### PR TITLE
addded basic dir structure

### PR DESCRIPTION
**Pull request by aagavin**

### CODE REVIEW

**Comment by aagavin** [comment]
This closes #5

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+from setuptools import find_packages, setup
+
+setup(name="KeanuFromBehine",
```
Comment (aramrattan) [inline setup.py]: Spelling Error on name should be KeanuFromBehind

File: keanu/routes/customer.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint, jsonify
+
+customer_api = Blueprint('customer_api', __name__)
+
+@customer_api.route('/customer')
+def customer() -> dict:
+    """
+    """
+    return jsonify({})
```
Comment (aramrattan) [inline keanu/routes/customer.py]: spacing at end of file

File: .gitignore [hunk 3]
```diff
@@ -87,3 +87,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# idea files
+.idea
```
Comment (aramrattan) [inline .gitignore]: Spacing at end of file

File: setup.py (hunk 1 above)
Comment (aagavin) [inline setup.py]: fixed

File: keanu/routes/customer.py (hunk 2 above)
Comment (aagavin) [inline keanu/routes/customer.py]: fixed

File: .gitignore (hunk 3 above)
Comment (aagavin) [inline .gitignore]: fixed

**Pull request merged**